README.md

```diff
@@ -44,8 +44,8 @@ The r2 on the Lipophilicity dataset should be arround 0.731 +/- 0.029.
 ## Getting Started
 ### Extracting Molecular Descripotrs
 Run the script run_cddd.py to extract molecular descripotrs of your provided SMILES:
-```
-run_cddd.py --input smiles.smi --output descriptors.csv  --smiles_header smiles
+```bash
+cddd --input smiles.smi --output descriptors.csv  --smiles_header smiles
```

Comment:
Here's the new way for calling the python script. It now works sort of like any other command in the console.